### PR TITLE
File 455 fixed it:test exceptions

### PR DESCRIPTION
**Pull request by YuanZHANG910**

### CODE REVIEW

File: it/uk/gov/hmrc/fileupload/support/FakeFileUploadBackend.scala [hunk 1]
```diff
@@ -31,12 +31,11 @@ trait FakeFileUploadBackend extends BeforeAndAfterAll with ScalaFutures {
   final lazy val fileUploadBackendBaseUrl = s"http://localhost:$backendPort"
 
   override def beforeAll() = {
-    super.beforeAll()
```
Comment (kastoestoramadus) [inline it/uk/gov/hmrc/fileupload/support/FakeFileUploadBackend.scala]: Why it is not needed?

Comment (YuanZHANG910) [reply it/uk/gov/hmrc/fileupload/support/FakeFileUploadBackend.scala]: i have removed beforeAll, as when the object runs all related code run anyway there is no need to do beforeAll, but afterAll is needed.

File: it/uk/gov/hmrc/fileupload/support/FakeFileUploadBackend.scala [hunk 2]
```diff
@@ -8,53 +8,52 @@ import com.github.tomakehurst.wiremock.verification.LoggedRequest
...
 import play.api.http.Status
 import uk.gov.hmrc.fileupload.{EnvelopeId, FileId}
 
 import scala.collection.JavaConverters._
+import scala.concurrent.ExecutionContextExecutor
 
 trait FakeFileUploadBackend extends BeforeAndAfterAll with ScalaFutures {
   this: Suite =>
 
+  implicit override val patienceConfig: PatienceConfig = PatienceConfig(timeout = Span(500, Seconds), interval = Span(500, Millis))
```
Comment (kastoestoramadus) [inline it/uk/gov/hmrc/fileupload/support/FakeFileUploadBackend.scala]: 500 is far too long

File: it/uk/gov/hmrc/fileupload/support/FileActions.scala [hunk 3]
```diff
@@ -66,10 +63,10 @@ trait FileActions extends ActionsSupport {
         "X-Requested-With" -> "someId")
       .post(s"-----011000010111000001101001\r\n" +
         "Content-Disposition: form-data; name=\"file1\"; filename=\"test.pdf\"\r\n" +
-        "Content-Type: application/pdf\r\n\r\n" +
+        "Content-Type: application/pdf\r\n" +
```
Comment (kastoestoramadus) [inline it/uk/gov/hmrc/fileupload/support/FileActions.scala]: why we had/have \r\n ? It look like windows new line.

Comment (YuanZHANG910) [reply it/uk/gov/hmrc/fileupload/support/FileActions.scala]: "\r\n" was there for sometimes now, i believe that is the way to create a text file, as without "\r\n" file will not created

**Comment by YuanZHANG910** [comment]
updated with @kastoestoramadus 's feedbacks.

File: it/uk/gov/hmrc/fileupload/support/FakeFileUploadBackend.scala [hunk 4]
```diff
@@ -8,53 +8,52 @@ import com.github.tomakehurst.wiremock.verification.LoggedRequest
...
 import play.api.http.Status
 import uk.gov.hmrc.fileupload.{EnvelopeId, FileId}
 
 import scala.collection.JavaConverters._
+import scala.concurrent.ExecutionContextExecutor
 
 trait FakeFileUploadBackend extends BeforeAndAfterAll with ScalaFutures {
   this: Suite =>
 
+  implicit override val patienceConfig: PatienceConfig = PatienceConfig(timeout = Span(5, Seconds), interval = Span(1, Seconds))
```
Comment (kastoestoramadus) [inline it/uk/gov/hmrc/fileupload/support/FakeFileUploadBackend.scala]: interval of 1 second is too long

File: it/uk/gov/hmrc/fileupload/support/FileActions.scala [hunk 5]
```diff
@@ -1,13 +1,27 @@
...
 
 import org.scalatest.Suite
-import org.scalatest.time.{Millis, Seconds, Span}
 import play.api.libs.ws.WSResponse
 import uk.gov.hmrc.fileupload.{EnvelopeId, FileId}
 
 trait FileActions extends ActionsSupport {
   this: Suite =>
 
+  val file: String = "-----011000010111000001101001\r\n" +
```
Comment (kastoestoramadus) [inline it/uk/gov/hmrc/fileupload/support/FileActions.scala]: refactor it to be as tidy as this:
https://gist.github.com/EECOLOR/7649144

Comment (YuanZHANG910) [reply it/uk/gov/hmrc/fileupload/support/FileActions.scala]: done

**Pull request merged**